desktop/appcontainer/ToastSource.js

```diff
@@ -80,8 +83,6 @@ class LocalModel extends HoistModel {
             container = containerRef ? containerRef : document.body,
             className = `xh-toast-container ${containerRef ? 'xh-toast-container--anchored' : ''}`;
 
-        position = withDefault(position, 'top');
```

Comment:
I thought it made more sense to default the position with the `ToastModel` ctor, which I did. 

Given that this entire class is doc'd as `@private` I am assuming it's safe the change the behavior of this method.